### ISSUE TITLE
Custom value for axes

### ISSUE THREAD

**Issue by jetsonwork** [marcomusy/vedo#956]
Hello
I am working on a 3D plot in vedo, where the height of the object is based on centimeters and has a range from 0 to 4000 cm. I want to change the values of the Z-axis to represent the height in meters instead, so the range should be from 0 to 4, where each unit on the Z-axis represents 10 meters. How can I achieve this in vedo?

Thank you

**Comment by marcomusy** [comment]
Yes it is possible, but you need to keep in mind that it is just a representation e.i. you change the appearance of the axes not the data associated to 3d world coordinates system which cannot be changed.
You can find examples in
`examples/pyplot/custom_axes1.py`
You basically need to specify
`y_values_and_labels=...`
